index.js

```diff
@@ -156,7 +156,15 @@ function wrap(content, config, packageRoot, browserAliases) {
   return boundWindow;
 }
 
-module.exports = function shim(file) {
+module.exports = shim;
+function shim(shimConfig$file , file) {
+
+  //check param type;
+  if(arguments.length===2){
```

Comment:
Check for `file` instead of args length here, i.e. 

```js
if (!file) {
  // handle case of one arg figure out what's what here
} else {
  // handle case of two args
}
```